source/write_docs/content/index.html.md.erb

```diff
@@ -22,27 +22,30 @@ Use Markdown when there are no specific formatting requirements.
 
 This example is a left-aligned table.
 
-You should include `<div style="height:1px;font-size:1px;">&nbsp;</div>` before and after each table.
-
-```bash
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-```
 ```bash
-|Column header|Column header|
-|:---|:---|
-|content|content|
-|content|content|
-```
-```bash
-<div style="height:1px;font-size:1px;">&nbsp;</div>
+| Column header | Column header |
+|---------------|---------------|
+| content       | content       |
+| content       | content       |
 ```
 
-You can change the alignment of the text:
+Add more columns and rows as needed.
+
+You can also change the alignment of the text in each column:
 
 - Centre: `|:---:|`
 - Right: `|---:|`
 
-Add more columns and rows as needed.
+If a table has three or more columns, you likely need to set a row header in addition to the column headers. Setting a row header improves the experience for screen reader users, as it reduces the amount of information they need to remember while navigating the table.
```

Comment:
- Style point: "three" => "3".
- Recommend rephrasing "you likely need to" to "you might need to" 
- Recommend adding a link to the W3 accessibility tutorials on tables at https://www.w3.org/WAI/tutorials/tables/ for more information to help people decide if they need to add a row header.

Reply:
Thanks for the review 👍 I've pushed up an amended commit.